Sources/RichTextKit/Actions/RichTextAction.swift

```diff
@@ -158,6 +161,7 @@ public extension RichTextAction {
         case .stepSuperscript(let steps): .actionStepSuperscript(steps)
         case .toggleStyle(let style): style.titleKey
         case .undoLatestChange: .actionUndoLatestChange
+        case .link: .actionCopy // TODO: Link
```

Comment:
Add this image before merging, just add a `link` image that uses the "link" symbol.